modulefiles/ufs_gaea.intel.lua

```diff
@@ -5,13 +5,7 @@ help([[
 
 whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea ]===])
 
-load("PrgEnv-intel/8.3.3")
-load("intel-classic/2023.1.0")
-load("cray-mpich/8.1.25")
-load("python/3.9.12")
```

Comment:
@SamuelTrahanNOAA these loads are removed from the modulefile for gaea, I would guess that one of these needs to be loaded still to avoid the errors you're seeing.

Reply:
I initially thought the same thing then checked the output for modules loaded (see below). Python is the only non-matching module. I'll try python/3.9.12 and report back...
```
Currently Loaded Modules:
  1) craype-x86-rome                        30) hdf5/1.14.0
  2) perftools-base/23.03.0                 31) snappy/1.1.10
  3) xpmem/2.6.2-2.5_2.38__gd067c3f.shasta  32) zstd/1.5.2
  4) cray-pmi/6.1.10                        33) c-blosc/1.21.4
  5) darshan-runtime/3.4.0                  34) curl/7.66.0
  6) CmrsEnv/default                        35) netcdf-c/4.9.2
  7) TimeZoneEDT/default                    36) netcdf-fortran/4.6.0
  8) DefApps/default                        37) parallel-netcdf/1.12.2
  9) cray-dsmml/0.2.2                       38) parallelio/2.5.10
 10) cray-libsci/23.02.1.1                  39) esmf/8.5.0
 **11) PrgEnv-intel/8.3.3**                     40) fms/2023.02.01
 **12) intel-classic/2023.1.0**                 41) bacio/2.4.1
 13) craype/2.7.20                          42) crtm-fix/2.4.0_emc
 14) stack-intel/2023.1.0                   43) git-lfs/2.11.0
 15) craype-network-ofi                     44) crtm/2.4.0
 **16) cray-mpich/8.1.25**                      45) g2/3.4.5
 17) stack-cray-mpich/8.1.25                46) g2tmpl/1.10.2
 18) gettext/0.20.2                         47) ip/4.3.0
 19) libxcrypt/4.4.35                       48) sp/2.3.3
 20) sqlite/3.42.0                          49) w3emc/2.10.0
 21) util-linux-uuid/2.38.1                 50) gftl/1.10.0
 **22) python/3.10.8**                          51) gftl-shared/1.6.1
 23) stack-python/3.10.8                    52) fargparse/1.5.0
 24) cmake/3.23.1                           53) mapl/2.40.3-esmf-8.5.0
 25) libjpeg/2.1.0                          54) scotch/7.0.4
 26) jasper/2.0.32                          55) ufs_common
 27) zlib/1.2.13                            56) nccmp/1.9.0.1
 28) libpng/1.6.37                          57) modules.fv3
 29) pkg-config/0.29.2
```

Reply:
`python/3.9.12` still produces the same original error.  For completeness I copied that working directory to: `/gpfs/f5/ufs-ard/world-shared/for_sam/rt_65400`